docs/source/flow-project.rst

```diff
@@ -214,6 +214,14 @@ As shown before, all *eligible* operations can then be executed with:
 
     ~/my_project $ python project.py run
 
+The status determination is by default parallelized with threads, however this can be turned off or switched to using processes by setting a value for the ``flow.status_parallelization`` configuration key to either ``none``, ``processes`` with ``threads`` being the implicit default value.
+
+You can set the ``flow.status_parallelization`` configuration value by using the command below where the valid values of the attribute ``VALUE`` are given above:
+
+.. code-block:: bash
+
+    ~/my_project $ signac config set flow.status_parallelization VALUE
```

Comment:
```suggestion
    ~/my_project $ signac config set flow.status_parallelization process
```
I suggest that we show directly one of the two sensible command as example. I find that it is usually much more helpful for users to see the *actual* command instead of a templated one that they still need to adapt.

Reply:
Yes, I agree. I'll edit.